src/core/rendering.js

```diff
@@ -146,6 +146,11 @@ p5.prototype.resizeCanvas = function (w, h, noRedraw) {
     // save canvas properties
     var props = {};
     for (var key in this.drawingContext) {
+      if (key === 'webkitImageSmoothingEnabled') {
```

Comment:
since it's deprecated, it seems like we culd just ignore this key completely and continue after this line?